client/src/pages/Home.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import styled from "@emotion/styled";
+
+const Container = styled.div`
+  background-color: #f2f2f2;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  > p {
+    margin: 0;
+  }
+`;
```

Comment:
It's the same container as in `Welcome`. If you want to have this container on every page, you could move it to your `App`.

Reply:
I will add this in global styling later. Right now it's only a reference with some basic details